corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -1,67 +1,77 @@
-hqDefine("scheduling/js/conditional_alert_main", [
```

Comment:
Could you revert the non-meaningful changes in this commit? I think that's everything except for removing `newRuleName`.

Reply:
thank you!